debian/kolibri-server.init

```diff
@@ -45,7 +48,7 @@ KOLIBRI_USER_HOME="$(getent passwd $KOLIBRI_USER | awk -F ':' '{print $6}')"
 DAEMON_HOME="$KOLIBRI_USER_HOME/.kolibri"
 
 DAEMON_UWSGI_ARGS="--ini /etc/kolibri/dist/uwsgi.ini --uid=$KOLIBRI_USER \
-  --env=KOLIBRI_HOME=$DAEMON_HOME --daemonize=$DAEMON_HOME/uwsgi.log --pidfile=$PIDFILE_UWSGI"
+  --env=KOLIBRI_HOME=$DAEMON_HOME --daemonize=$DAEMON_HOME/uwsgi.log --pidfile2=$PIDFILE_UWSGI"
```

Comment:
pidfile2?

Reply:
Yes, it's a terrible name, I don't get why they chose this convention.

https://uwsgi-docs.readthedocs.io/en/latest/Options.html#pidfile2

https://unix.stackexchange.com/a/121478/128349

Reply:
I was about to write some swearword to express my opinion about that api, but that would not solve anything. Thanks for the catch.